homeassistant/components/vicare/climate.py

```diff
@@ -85,13 +86,15 @@
 VICARE_TO_HA_PRESET_HEATING = {
     VICARE_PROGRAM_COMFORT: PRESET_COMFORT,
     VICARE_PROGRAM_ECO: PRESET_ECO,
-    VICARE_PROGRAM_NORMAL: PRESET_NONE,
+    VICARE_PROGRAM_NORMAL: PRESET_HOME,
+    VICARE_PROGRAM_REDUCED: PRESET_SLEEP,
 }
 
 HA_TO_VICARE_PRESET_HEATING = {
     PRESET_COMFORT: VICARE_PROGRAM_COMFORT,
     PRESET_ECO: VICARE_PROGRAM_ECO,
-    PRESET_NONE: VICARE_PROGRAM_NORMAL,
+    PRESET_HOME: VICARE_PROGRAM_NORMAL,
+    PRESET_SLEEP: VICARE_PROGRAM_REDUCED,
```

Comment:
This results in having the presets selectable via select or button. But actually these presets are determined via the internal heating schedule. So they cannot be activated. When selecting the "home" during night time, there will be no error, but the preset will fallback to "reduced".

Is this behavior acceptable? I would like to see which preset is active (like seen on the device page).
<img width="331" alt="Bildschirmfoto 2023-12-13 um 11 19 04" src="https://github.com/home-assistant/core/assets/9592452/e4ee2149-c99d-4eaa-a79f-ff403d989174">

Reply:
Opened a feature discussion on that topic

https://github.com/home-assistant/frontend/discussions/19037

Reply:
Euh I don't really see what problem you are pointing at, could you maybe elaborate a bit?

Reply:
Viessmann heatings are always running on a schedule, *normal* (day) and *reduced* (night) that the user can define on its own. 
These are presets that are determined by the schedule only, one cannot activate/deactivate them. One can only overrule it with the other presets like *eco*, *comfort*.

Reply:
So there are some presets that can be used and some that can not?

Reply:
Hmm, so what will we do in this PR? Add the presets so they will show?

What if we set it to a preset you can't select?

Reply:
I would also be fine with removing this change and just go with the changes from line 88, so I can see the preset on the entity.

Reply:
> What if we set it to a preset you can't select?

Integration will ignore *reduced* and *normal*. So when you select *normal* during night time it will just do nothing and flip back to *reduced*.